style.css

```diff
@@ -5,6 +5,11 @@
   --text-color: #ffffff;
   --tweet-button-color: #1a91da;
   --item-bg-hover-color: #333b45;
+  --trends-bg-hover-items-color: #202F3A;
+  --trend-hashtag-tweets-color: #6D7B85;
+  --trend-more-info-color: #1DA1F2;
+  --trend-more-info-bg-color:#20394D;
```

Comment:
it is bad to name the variable with something related to content, please remove the trends word from the variables and replace it with something more generic